bridge/src/mock.rs

```diff
@@ -517,3 +517,16 @@ pub fn assert_events(mut expected: Vec<RuntimeEvent>) {
 		assert_eq!(next, evt, "Events don't match");
 	}
 }
+
+pub fn slice_to_generalkey(key: &[u8]) -> Junction {
+	let len = key.len();
+	assert!(len <= 32);
+	GeneralKey {
+		length: len as u8,
+		data: {
+			let mut data = [0u8; 32];
+			data[..len].copy_from_slice(key);
+			data
+		},
+	}
+}
```

Comment:
`generealKey` has `length` and `data` in v0.9.39 ? will this impact how relayer and SDK side when interacting with pallets? because we are using generalKey inside MultiLocation of token

Reply:
also, the max length of key is 32?

Reply:
ok got it